app/views/grants/_review_guidance.html.haml

```diff
@@ -0,0 +1,11 @@
+/ .nested-fields
+/   .field
+.grid-x.grid-padding-y
+  .cell
+    %h4
+      = f.label :review_guidance, 'Review Guidance'
+    %p
+      %em
+        You may provide instructions and guidelines to help them score the submissions.
```

Comment:
maybe:
"Optionally provide instructions and guidelines to help reviewers score the submission."